src/nav/codestream.yml

```diff
@@ -47,7 +47,7 @@ pages:
         path: /docs/codestream/troubleshooting/git-issues
       - title: GitHub organizations
         path: /docs/codestream/troubleshooting/github-org-repos
-      - title: GitLab Self-Managed version
+      - title: GitLab self-managed version
```

Comment:
@zuluecho9 - "GitLab Self-Managed" is a product name and needs to be capitalized.

Reply:
i saw it lower case here and elsewhere: https://docs.gitlab.com/ee/subscriptions/self_managed/. are you seeing it capitalized elsewhere? @planteater

Reply:
Looks like GL is inconsistent in their display, although that feels like a recent thing. I think we should stick with upper either way though, as we use that all throughout the product. And it's simply clearer that it's a product name.

Reply:
ok